domain/src/main/java/com/example/domain/Turn.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.domain
+
+@JvmInline
+value class Turn(private val value: Int) {
+    init {
+        require(value in 0..8) {
```

Comment:
턴이 9개를 넘어가지 않도록 하는 것도 판의 크기와 관련이 있네요..!
```suggestion
        require(value in 0 until BOARD_SIZE) {
```
와 같이 판의 크기를 상수화하고 until을 이용하는 것이 조금 더 직관적으로 보여줄 수 있다고 느껴집니다~